Sources/Public/BacktraceClientCustomizing.swift

```diff
@@ -6,12 +6,23 @@ public typealias BacktraceClientProtocol = BacktraceReporting & BacktraceClientC
 /// Type-alias of passing attributes to library.
 public typealias Attributes = [String: Any]
 
+/// Type-alias of passing file attachments to library.
+/// Expected format: Filename, File URL bookmark
+public typealias Attachments = [String: URL]
+
+/// Type-alias of storing file attachments on disk (as a bookmark)
+/// Expected format: Filename, File URL bookmark
+public typealias Bookmarks = [String: Data]
```

Comment:
Does it need to be public?

Reply:
@krml19 if it's private I won't be able to use it in my `AttachmentBookmarkHandlerMock` class